strapi/strapi
# enhancement(database)!: shorten prefixes and suffixes used in identifiers

### What does it do?

Use shorter prefixes and suffixes in generated metadata, for example using "cmps" instead of "components" and "lnk" instead of "links"

### Why is it needed?

To make more space available for the actual names of content types and attribute in our database names

### How to test it?

See https://github.com/strapi/strapi/pull/19732 and follow the same testing instructions. The only difference is that now suffixes and prefixes should also be renamed to the shorter versions.

### Related issue(s)/PR(s)

The core migration without shorter prefixes and suffixes: https://github.com/strapi/strapi/pull/19732
